include/kodlab_mjbots_sdk/filter.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file filter.h
+ * @author Lucien C. Peach (peach@seas.upenn.edu)
+ * @brief Abstract base class for support of filter implementation. 
+ * @date 9/10/2022
+ * 
+ * @copyright 2022 The Trustees of the University of Pennsylvania. All rights reserved.
+ * 
+ */
+
+#pragma once
+
+#include "Eigen/Geometry"
+
+namespace kodlab
+{
+
+class Filter {
+
+    public: 
+
+        // Pure virtual function for filter update calls
+        virtual Eigen::Vector3f Update(Eigen::Vector3f raw_data) = 0;
+
+        // Accessor for raw_data_
+        Eigen::Vector3f GetRawData() {
```

Comment:
**[General]** [Accessors and mutators (getters and setters) should be named in snake-case](https://drake.mit.edu/styleguide/cppguide.html#Function_Names).
```suggestion
        Eigen::Vector3f get_raw_data() {
```